package/src/components/Channel/hooks/useCreateInputMessageInputContext.ts

```diff
@@ -134,7 +134,15 @@ export const useCreateInputMessageInputContext = <
       UploadProgressIndicator,
     }),
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [compressImageQuality, channelId, editingDep, initialValue, maxMessageLength, quotedMessageId],
+    [
+      compressImageQuality,
+      channelId,
+      editingDep,
+      initialValue,
+      maxMessageLength,
+      quotedMessageId,
+      sendMessage,
```

Comment:
This will cause rerenders since `sendMessage` is defined inline. The correct way to do it would be to wrap `sendMessage` inside a `useCallback`, but there are many other dependencies that are also defined inline which will also need to be wrapped inside a `useCallback` or similar. Hence, we're just adding it as a dependency here and relying on that.